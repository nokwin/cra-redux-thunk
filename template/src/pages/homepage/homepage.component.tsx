import React from "react";
import { Helmet } from "react-helmet";

export const HomePage = () => (
  <>
    <Helmet>
      <title>Homepage</title>
    </Helmet>
    <h1>This is the homepage</h1>
  </>
);
