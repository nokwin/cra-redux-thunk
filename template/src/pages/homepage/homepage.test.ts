import { HomePage } from "./homepage.component";

describe("Homepage component", () => {
  it("should render homepage component", () => {
    expect(HomePage).toMatchSnapshot();
  });
});
