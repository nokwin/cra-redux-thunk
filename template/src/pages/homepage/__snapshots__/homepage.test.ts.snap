// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage component should render homepage component 1`] = `[Function]`;
